identity-concentrator/outputs.tf

```diff
@@ -12,6 +25,6 @@ output "cloudsql_root_user_password" {
 }
 
 output "cloudsql_app_db_creds" {
-  value = module.cloudsql.app_db_creds
+  value = length(module.cloudsql.app_db_creds) == 0 ? {} : module.cloudsql.app_db_creds[var.service]
```

Comment:
I ran into a TF error when trying to delete the resources. It looks like it tried to evaluate the output after the resource was deleted. This is to avoid that error when running `terraform destroy`
Agreed, will add comment.